packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -221,3 +227,19 @@ export const isDisabled = (option?: boolean | { disabled: boolean }): boolean =>
   }
   return option;
 };
+
+/**
+ * Check if an object is identifier.
+ *
+ * @param identifier
+ * @returns whether an identifier is one of identifier types (for runtime validation)
+ * @private
+ */
+export const isValidIdentifier = (identifier: CommunicationIdentifier): boolean => {
```

Comment:
Define in a common place so all adapters can use it without cross-importing from another composite's directory.